pkg/volume/azure_dd/azure_dd.go

```diff
@@ -106,6 +111,22 @@ func (plugin *azureDataDiskPlugin) SupportsBulkVolumeVerification() bool {
 	return false
 }
 
+func (plugin *azureDataDiskPlugin) GetVolumeLimits() (map[string]int64, error) {
+	cloud := plugin.host.GetCloudProvider()
+	if cloud.ProviderName() != azure.CloudProviderName {
+		return nil, fmt.Errorf("Expected Azure cloudprovider, got %s", cloud.ProviderName())
+	}
+
+	volumeLimits := map[string]int64{
```

Comment:
Question: @gnufied it looks like I need to write code here to query volume limits according to azure node type , and assign new value to `volumeLimits ` in `azure_dd.go`, right?

Reply:
yes that is correct. I did not know how we can get those limits for Azure Disk and hence I went with existing defaults.